linker/linker_test.go

```diff
@@ -2604,25 +2617,25 @@ func TestLinkerValidation(t *testing.T) {
 				"test.proto": `
 					syntax = "proto2";
 					message A {
-						extensions 100 to 500, 1000 to 5000 [
+						extensions 1000 to 5000 [
 							verification=DECLARATION,
 							declaration={
-								number: 501
+								number: 5001
 								full_name: ".foo.bar"
 								type: "int32"
 							}
 						];
 					}
 				`,
 			},
-			expectedErr: `test.proto:6:25: extension declaration has number outside the range: 501 not in [100,500]`,
+			expectedErr: `test.proto:6:25: extension declaration has number outside the range: 5001 not in [1000,5000]`,
 		},
 		"failure_extension_declaration_with_number_out_of_range_multiple_ranges": {
 			input: map[string]string{
 				"test.proto": `
 					syntax = "proto2";
 					message A {
-						extensions 1 to 3, 5 to 10, 100 to 500 [
+						extensions 1 to 3, 5 to 10 [
```

Comment:
A single non-applicable range sufficed. So instead of updating `expectedErr` to show multiple errors, I just tweaked this test case.